web/src/components/tasks/business-formation/contacts/ContactsStep.tsx

```diff
@@ -111,7 +119,9 @@ export const ContactsStep = (): ReactElement => {
               snackbarBody: Config.formation.fields.incorporators.successSnackbarBody,
               modalTitle: Config.formation.fields.incorporators.modalTitle,
               modalSaveButton: Config.formation.fields.incorporators.addButtonText,
+              error: Config.formation.fields.incorporators.error,
             }}
+            hasError={doesFieldHaveError("signers") || doesFieldHaveError("incorporators")}
```

Comment:
[dust] 
So this line confuses me a bit. 

It looks to me (at a glance) that the "signers" applies to the <Signatures/> component and the <Addresses<FormationIncorporator> component applies to "incorporators".

So I'm confused as to why this field specifically needs both the Error status of "signers" and "incorporators". Can you clarify for me?

Beyond that I was thinking if it only applied to "incorporators" have the fieldName passed here we could call doesFieldHaveError in the child component and have one fewer prop

Reply:
incorporators is for corporations, signers are for all other business structures